system-tests/test/pause_headed_exit_spec.ts

```diff
@@ -0,0 +1,13 @@
+import systemTests from '../lib/system-tests'
+
+describe('cy.pause() works with --headed and --no-exit', () => {
+  systemTests.setup()
+
+  systemTests.it('passes', {
+    spec: 'pause_simple_spec.js',
+    snapshot: false,
+    headed: true,
+    noExit: true,
```

Comment:
Somewhat surprised a test can have an exit code of 0 **and** have `noExit` as true - seems like these would conflict, but seems this isn't the case?

Reply:
At least locally this seems to hang forever (on develop, was verifying it fails there)

![image](https://user-images.githubusercontent.com/19196536/162099936-8d6d3ea4-0029-480f-9c31-eab91f24f7b2.png)

Reply:
Yes this test is still a wip- it will hang forever because `--no-exit` has been passed

Reply:
Currently working on about adding a callback to ExecOptions (something like onSpawn) that yields the ChildProcess so we can listen to the stdout and kill the ChildProcess within the system test itself when `not exiting due to options.exit being false` gets logged.

Reply:
Gotcha, can you mark it as draft? I can re-review when you are happy with it - I thought this was ready for review.

![image](https://user-images.githubusercontent.com/19196536/162100504-fceff166-4208-4bad-a1e8-3951640dc6be.png)